src/cmake/AxomOptions.cmake

```diff
@@ -10,7 +10,7 @@ option(AXOM_ENABLE_ANNOTATIONS "Enables code annotations to facilitate performan
 option(AXOM_ENABLE_SPARSEHASH "Enables Sparsehash." ON)
 option(AXOM_ENABLE_ALL_COMPONENTS "Enables all components by default" ON)
 option(AXOM_USE_64BIT_INDEXTYPE "Use 64-bit integers for axom::IndexType" OFF)
-option(AXOM_ENABLE_MFEM_SIDRE_DATACOLLECTION "Enable Axom's version of the MFEM SidreDataCollection" OFF)
+option(AXOM_ENABLE_MFEM_SIDRE_DATACOLLECTION "Enable Axom's version of the MFEM SidreDataCollection" ON)
```

Comment:
I'd like to explicitly call out this change -- now that we have parallel mfem, I'm enabling `AXOM_ENABLE_MFEM_SIDRE_DATACOLLECTION` by default.
If anyone is building mfem w/ `MFEM_USE_SIDRE`, they'll want/need to turn this option off.